autogpt_platform/backend/backend/data/graph.py

```diff
@@ -353,38 +353,46 @@ async def get_node(node_id: str) -> Node:
 async def get_graphs(
     user_id: str,
     include_executions: bool = False,
-    filter_by: Literal["active", "template"] | None = "active",
+    filter_by: Literal["all", "active", "template"] | None = "active",
 ) -> list[Graph]:
     """
     Retrieves graph metadata objects.
     Default behaviour is to get all currently active graphs.
+    When filter_by="all", retrieves all versions of graphs.
```

Comment:
I'm pretty sure we have bajillions of graphs without an execution because of the way we save the graph. Should we have a filter to fetch all graphs with at least a single execution instead? Or even better, the fetch is done using `get_executions` (which fetches all graph executions for a user, I think we don't have it yet, but it should be easy to add) instead of `get_graph` and the graph we are fetching is still only the active ones.